### ISSUE TITLE
[Feature Request]: Create a small admin panel for alerts

### ISSUE THREAD

**Issue by ericboucher** [WFP-VAM/prism-frontend#1304]
### Provide a clear and concise description of what you want to happen.

Following WFP IT rules, the alerts database is not accessible from the web, which makes it hard to manually access it to look at data and update/delete alerts. I think it would be useful to add a few secured routes to the API allowing admins to see existing alerts and run basic update/delete operations

**Comment by wadhwamatic** [comment]
We've also recently developed a few marimo apps that interact with AWS sign-on that might do the job. cc @valpesendorfer

**Comment by ericboucher** [comment]
cc @gislawill @c-mac

**Comment by gislawill** [comment]
Noting overlap with #1769.

**Comment by c-mac** [comment]
Yep - plan is to just expose views to the alerts tables in `starlette-admin`
